libraries/botbuilder-middleware-aws-comprehend/src/engine.ts

```diff
@@ -0,0 +1,128 @@
+import { Engine } from '@botbuildercommunity/middleware-engine-core';
+import * as aws from 'aws-sdk';
+
+/**
+ * @module botbuildercommunity/middleware-aws-comprehend
+ */
+
+export class AWSComprehendEngine extends Engine {
+    public client: aws.Comprehend;
+    public constructor() {
+        super();
+        this.client = new aws.Comprehend({ apiVersion: '2017-11-27', region: 'us-east-1' });
+    }
+    public async entities(input: any): Promise<any> {
+        const params = {
+            LanguageCode: 'en',
```

Comment:
Why is languageCode hardcoded? Shouldn't this be based on the language code of the activity or overriden by the config.

Reply:
Added to `AWSComprehendOptions`.